### PR TITLE
Hyper Func

### PR DESCRIPTION
**Pull request by imeoer**

### CODE REVIEW

File: html/pricing/main.handlebars [hunk 1]
```diff
@@ -54,6 +54,14 @@
         </div>
       </li>
       <li class="section">
+        <div class="title">> Hyper Func (BETA)</div>
+        <div class="content">
+          <ul>
+            <li>During the beta, Hyper Func is charged at <code>the standard per-second container rates * 100%</code>.</li>
```
Comment (mrmrcoleman) [inline html/pricing/main.handlebars]: What does this mean? "*100%"

File: html/features/main.handlebars [hunk 2]
```diff
@@ -51,6 +51,13 @@
               <div class="subtitle">Schedule serverless cron jobs using Docker images by simply <code>hyper cron create</code>.</div>
             </div>
           </li>
+          <li class="item">
+            <div class="icon-wrap"><img src="/assets/icons/features/func.svg" class="icon" alt="Effortless Docker Hosting, Docker, Container"></div>
+            <div class="info">
+              <div class="title">Hyper Func (BETA)</div>
+              <div class="subtitle">A docker-centric serverless platform.</div>
```
Comment (mrmrcoleman) [inline html/features/main.handlebars]: A Docker-centric serverless platform. (Docker is with a capital D)

File: html/pricing/main.handlebars (hunk 1 above)
Comment (mrmrcoleman) [inline html/pricing/main.handlebars]: I still didn't get an answer to this question.

Comment (mrmrcoleman) [inline html/pricing/main.handlebars]: During the beta, Hyper Func is charged at <code>the standard per-second conatiner rates</code>

File: html/pricing/main.handlebars [hunk 3]
```diff
@@ -54,6 +54,14 @@
         </div>
       </li>
       <li class="section">
+        <div class="title">> Hyper Func (BETA)</div>
+        <div class="content">
+          <ul>
+            <li>During the beta, Hyper Func is charged at the standard per-second conatiner rates.</li>
```
Comment (kevinsimper) [inline html/pricing/main.handlebars]: conatiner is spelled wrong 🙂

**Pull request merged**